nav_msgs/OccupancyGrid[] maps
geometry_msgs/Pose[] origins
